Gemfile.lock

```diff
@@ -388,4 +388,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```

Comment:
I *believe* this change means we may need to `gem install bundler -v 2.0.2` on all the prescat stage and prod instances in advance of our next round of deployments.

Reply:
we'll find out on Monday, no doubt.